bin/hdfcoinc/pycbc_fit_sngls_binned

```diff
@@ -35,14 +35,18 @@ stat_dict = {
     "new_snr"       : events.newsnr,
     "effective_snr" : events.effsnr,
     "snr"           : lambda snr, rchisq : snr,
-    "snronchi"      : lambda snr, rchisq : snr / (rchisq ** 0.5)
+    "snronchi"      : lambda snr, rchisq : snr / (rchisq ** 0.5),
+    "newsnr_sgveto"      : events.newsnr_sgveto,
 }
 
-def get_stat(statchoice, snr, rchisq, fac):
+def get_stat(statchoice, snr, rchisq, sgchisq, fac):
+    if statchoice not in stat_dict:
+        raise RuntimeError("Can't use --stat-factor with this statistic!")
```

Comment:
this error message now makes no sense.  if statchoice is not one of the allowed statistics, the problem is nothing to do with the stat-factor argument